src/lens_distortion_correction_2p_iterative_optimization.cpp

```diff
@@ -386,20 +396,27 @@ int main(int argc, char *argv[])
                             height
                           );
     
-    cout << "...lines detected: " << i_primitives.get_lines().size() <<
-        " with " << count_points(i_primitives) << " points" << std::endl;
-    cout << "image_error: " << image_error << std::endl;
+    cout << "        lines detected: " << i_primitives.get_lines().size() << std::endl;
+    cout << "        nb points: " << count_points(i_primitives) << std::endl;
+    cout << "        image error: " << image_error << std::endl;
+    
     //We check if the iterative optimization process finishes properly
     if(i_primitives.get_lines().size() == 0)
     {
       manage_failure(argv,0);
       continue;
     }
+    ++count_frames; // one new frame successfully computed
+    
+    const int nbPoints = count_points(i_primitives);
+    const double score = nbPoints / std::max(0.5, image_error);

```

Comment:
Add comment:
```
// The score try to maximize the number of points with a weighting by the average error per point.
// The average error per point is bounded to the best error we can hope(0.5 pixel) to avoid large scores in extreme cases: few points can lead to a really small error.
```